registry/polkadot.json

```
@@ -1,4 +1,12 @@
 [
+     {
+        "address": "GqPDeJBexSsvEq6gBzrnMCn7KNYbzn9bX9ePCjEDN3EikoV",
```

Looks like address is mixed up.
```suggestion
        "address": "15G4hfDNtNhRc82As8Ep2YfvpM5xVdX7De3P9qSdHerGA6wC",
```